packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -599,11 +605,74 @@ abstract class ScrollView extends StatelessWidget {
 ///    the scroll position without using a [ScrollController].
 ///  * [IndexedSemantics], which allows annotating child lists with an index
 ///    for scroll announcements.
-class CustomScrollView extends ScrollView {
+///
+class CustomScrollView extends StatelessWidget {
   /// Creates a [ScrollView] that creates custom scroll effects using slivers.
   ///
   /// See the [ScrollView] constructor for more details on these arguments.
   const CustomScrollView({
+    this.key,
+    this.scrollDirection = Axis.vertical,
+    this.reverse = false,
+    this.controller,
+    this.primary,
+    this.physics,
+    this.shrinkWrap = false,
+    this.center,
+    this.anchor = 0.0,
+    this.cacheExtent,
+    this.slivers = const <Widget>[],
+    this.semanticChildCount,
+    this.dragStartBehavior = DragStartBehavior.start,
+    this.keyboardDismissBehavior = ScrollViewKeyboardDismissBehavior.manual,
+    this.restorationId,
+    this.clipBehavior = Clip.hardEdge,
+    this.scrollBehavior,
+  }) : super(key: key);
+
+  // TODO(Piinks): All of these need public docs now
+  final Key? key;
+  final Axis scrollDirection;
+  final bool reverse;
+  final ScrollController? controller;
+  final bool? primary;
+  final ScrollPhysics? physics;
+  final bool shrinkWrap;
+  final Key? center;
+  final double anchor;
+  final double? cacheExtent;
+  final List<Widget> slivers;
+  final int? semanticChildCount;
+  final DragStartBehavior dragStartBehavior;
+  final ScrollViewKeyboardDismissBehavior keyboardDismissBehavior;
+  final String? restorationId;
+  final Clip clipBehavior;
+  final ScrollBehavior? scrollBehavior;
+
+  @override
+  Widget build(BuildContext context) {
+    return _CustomScrollView(
+      key: key,
+      scrollDirection: scrollDirection,
+      reverse: reverse,
+      controller: controller,
+      primary: primary,
+      physics: physics ?? scrollBehavior?.getScrollPhysics(context),
+      shrinkWrap: shrinkWrap,
+      center: center,
+      anchor: anchor,
+      cacheExtent: cacheExtent,
+      semanticChildCount: semanticChildCount,
+      dragStartBehavior: dragStartBehavior,
+      keyboardDismissBehavior: keyboardDismissBehavior,
+      restorationId: restorationId,
+      clipBehavior: clipBehavior,
+      scrollBehavior: scrollBehavior ?? ScrollConfiguration.of(context).copyWith(scrollbarPlatforms: <TargetPlatform>{}),
```

Comment:
Why does this one not get scrollbars?

Reply:
CustomScrollView shouldn't get scrollbars because the Scrollbar cannot automatically handle whatever folks are building in the CustomScrollView, like avoiding overlapping SliverAppBars for example.